### PR TITLE
Update clusterpedia image to v0.5.2

### PR DESCRIPTION
**Pull request by Songjoy**
fix issues: https://github.com/clusterpedia-io/clusterpedia/issues/464

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/clusterpedia/clusterpedia-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/clusterpedia/clusterpedia-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/clusterpedia/clusterpedia-helm/blob/main/CONTRIBUTING.md).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

**Comment by calvin0327** [comment]
@Songjoy thank you so much for your contribution. 

Please bump chart version `+1`, for details please see  [versioning](https://github.com/clusterpedia-io/clusterpedia-helm/blob/main/CONTRIBUTING.md#versioning)

**Comment by Songjoy** [comment]
update @calvin0327

File: charts/clusterpedia/Chart.yaml [hunk 1]
```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.2
+version: 1.0.3
```
Comment (Iceber) [inline charts/clusterpedia/Chart.yaml]: This is not a bug fix, we can update to 1.1.0

**Review by Iceber — COMMENTED** [review]
Other looks good

File: charts/clusterpedia/Chart.yaml (hunk 1 above)
Comment (Songjoy) [reply charts/clusterpedia/Chart.yaml]: ok, done @Iceber

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**